test/test_support.ts

```diff
@@ -71,6 +71,14 @@ export function createProgramAndHost(
     }
     throw new Error('unexpected file read of ' + fileName + ' not in ' + toArray(sources.keys()));
   };
+  const originalFileExists = host.fileExists;
+  host.fileExists = (fileName: string): boolean => {
+    if (path.isAbsolute(fileName)) fileName = path.relative(process.cwd(), fileName);
+    if (sources.has(fileName)) {
+      return true;
+    }
```

Comment:
So that the `import` / `export` statements in the transformer_util_test.ts actually work. Otherwise I can't test that exports are elided because TypeScript cannot see the files.

We don't need this for the regular golden tests as TS can find them via the default implementation of the compiler host.